User.cc

```diff
@@ -0,0 +1,28 @@
+#include "User.h"
+#include "Portfolio.h"
+#include <memory>
+#include <string>
+#include <iostream>
+using namespace std;
+
+User::User(string username): username{username}, cashBalance{1000} {
+  Portfolio portfolio;
```

Comment:
no need to do this here again. The portfolio will have already been defaulted constructed